pkg/util/gc_service.go

```diff
@@ -23,15 +23,15 @@ import (
 
 const (
 	// cdcChangefeedCreatingServiceGCSafePointID is service GC safe point ID
-	cdcChangefeedCreatingServiceGCSafePointID = "ticdc-changefeed-creating"
+	cdcChangefeedCreatingServiceGCSafePointID = "ticdc-creating-"
```

Comment:
```suggestion
	cdcChangefeedCreatingServiceGCSafePointID = "_ticdc_lock"
```
Is this name better?